elements/nuxeo-document-info-bar/nuxeo-document-info-bar.html

```diff
@@ -224,7 +221,7 @@
         if (confirm(this.i18n('documentPage.abandon.workflow.confirm'))) {
           this.$.worfklow.path = '/workflow/' + e.model.workflow.id;
           this.$.worfklow.remove().then(function() {
-            this.fire('workflowAbandoned');
+            this.fire('workflowAbandoned', {workflow: e.model.workflow});
```

Comment:
Use space inside brackets:
```suggestion
            this.fire('workflowAbandoned', { workflow: e.model.workflow });
```